python/lsst/ctrl/bps/transform.py

```diff
@@ -231,6 +216,8 @@ def _enhance_command(config, generic_workflow, gwjob):
     gwjob : `lsst.ctrl.bps.GenericWorkflowJob`
         Generic workflow job to which the updated executable, arguments,
         and values should be saved.
+    cached_job_values : `dict` [`str`, `str`]
```

Comment:
Shouldn't the type description be `dict [str, Any]`? For example, in line 239 below you add `bpsUseShared` to `cached_job_values` which I assume will be a boolean value.